libutils/string_sequence.c

```diff
@@ -187,7 +187,9 @@ bool SeqStringWriteFile(Seq *seq, const char *file)
     {
         return false;
     }
-    return SeqStringWriteFileStream(seq, f);
+    bool success = SeqStringWriteFileStream(seq, f);
+    fclose(f);
```

Comment:
I guess we don't care about any write issues happening here?

Reply:
Fixed.